public/refs/paths/accounts_{account_id}_sites_{site_identifier}_subjects_{subject_number}.json

```diff
@@ -194,5 +194,105 @@
         "$ref": "../components/responses/InternalServerError.json"
       }
     }
+  },
+  "delete": {
+    "summary": "Delete a study subject for a site",
+    "description": "Delete a study subject for a site",
+    "operationId": "deleteSubject",
+    "tags": ["Study Subjects"],
+    "security": [
+      {
+        "userSessionKey": [],
+        "chillipharmSession": []
+      }
+    ],
+    "parameters": [
+      {
+        "name": "account_id",
+        "in": "path",
+        "description": "The account id",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "1"
+        }
+      },
+      {
+        "name": "site_identifier",
+        "in": "path",
+        "description": "The site identifier",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "SITE1"
+        }
+      },
+      {
+        "name": "subject_number",
+        "in": "path",
+        "description": "The subject number",
+        "required": true,
+        "schema": {
+          "type": "string",
+          "example": "12345678"
+        }
+      }
+    ],
+    "requestBody": {
+      "content": {
+        "application/json": {
+          "schema": {
+            "type": "object",
+            "properties": {
+              "authenticity_token": {
+                "$ref": "../components/schemas/AuthenticityToken.json"
+              },
+              "number": {
```

Comment:
we don't need the number in the body as it's already defined in the route url path